src/darwin/Framework/CHIP/MTRDeviceController+XPC.h

```diff
@@ -76,6 +76,15 @@ typedef void (^MTRValuesHandler)(id _Nullable values, NSError * _Nullable error)
  */
 + (MTRSubscribeParams * _Nullable)decodeXPCSubscribeParams:(NSDictionary<NSString *, id> * _Nullable)params;
 
+/**
+ * Returns an NSXPCInterface configured for MTRDeviceControllerServerProtocol.
+ */
++ (NSXPCInterface *)interfaceForServerProtocol MTR_NEWLY_AVAILABLE;
```

Comment:
`xpcInterfaceForServerProtocol` might be a little more descriptive?

Reply:
Yep, makes sense.